src/test/resources/saveScriptTest.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0"?><script><cameras><camera id="0"><cameraSettings pan="0" tilt="0" zoom="0" focus="1365"></cameraSettings><presets><preset type="main.java.nl.tudelft.contextproject.presets.InstantPreset"><id>1</id><description></description><imgLoc></imgLoc><cameraSettings pan="1" tilt="1" zoom="1" focus="1365"></cameraSettings></preset><preset type="main.java.nl.tudelft.contextproject.presets.InstantPreset"><id>4</id><description></description><imgLoc></imgLoc><cameraSettings pan="4" tilt="4" zoom="4" focus="1365"></cameraSettings></preset></presets></camera><camera id="1"><cameraSettings pan="0" tilt="0" zoom="0" focus="1365"></cameraSettings><presets><preset type="main.java.nl.tudelft.contextproject.presets.InstantPreset"><id>2</id><description></description><imgLoc></imgLoc><cameraSettings pan="2" tilt="2" zoom="2" focus="1365"></cameraSettings></preset><preset type="main.java.nl.tudelft.contextproject.presets.InstantPreset"><id>3</id><description></description><imgLoc></imgLoc><cameraSettings pan="3" tilt="3" zoom="3" focus="1365"></cameraSettings></preset></presets></camera><camera id="2"><cameraSettings pan="0" tilt="0" zoom="0" focus="1365"></cameraSettings><presets><preset type="main.java.nl.tudelft.contextproject.presets.InstantPreset"><id>1</id><description></description><imgLoc></imgLoc><cameraSettings pan="1" tilt="1" zoom="1" focus="1365"></cameraSettings></preset><preset type="main.java.nl.tudelft.contextproject.presets.InstantPreset"><id>4</id><description></description><imgLoc></imgLoc><cameraSettings pan="4" tilt="4" zoom="4" focus="1365"></cameraSettings></preset></presets></camera><camera id="3"><cameraSettings pan="0" tilt="0" zoom="0" focus="1365"></cameraSettings><presets><preset type="main.java.nl.tudelft.contextproject.presets.InstantPreset"><id>2</id><description></description><imgLoc></imgLoc><cameraSettings pan="2" tilt="2" zoom="2" focus="1365"></cameraSettings></preset><preset type="main.java.nl.tudelft.contextproject.presets.InstantPreset"><id>3</id><description></description><imgLoc></imgLoc><cameraSettings pan="3" tilt="3" zoom="3" focus="1365"></cameraSettings></preset></presets></camera></cameras><shots><shot number="0"><shotId></shotId><description></description><cameraId>2</cameraId><presetId>1</presetId></shot><shot number="1"><shotId></shotId><description></description><cameraId>3</cameraId><presetId>2</presetId></shot><shot number="2"><shotId></shotId><description></description><cameraId>3</cameraId><presetId>3</presetId></shot><shot number="3"><shotId></shotId><description></description><cameraId>2</cameraId><presetId>4</presetId></shot></shots></script>
```

Comment:
@TH3steven We have discussed this in our team's group chat on WhatsApp and came to the conclusion that this does not matter and might even be better to prevent humans from accidentally corrupting the save file while tinkering with it.